bin/jpm

```diff
@@ -74,7 +75,17 @@ program
   .option("--xpi <filePath>", "optional XPI to be signed. By default, an XPI will be built from your working directory")
   .option("--timeout <milliseconds>", "time to wait for a signing result")
   .action(function(options) {
-    sign(program, options);
```

Comment:
as mentioned earlier, I think if you put this back then the tests won't need so many changes